starboard/commands/override.py

```diff
@@ -271,15 +271,12 @@ async def callback(self, ctx: crescent.Context) -> None:
         ovd = ov.overrides
         c = 0
         for o in options:
+            c += 1
```

Comment:
This isn't quite right. `c+=1` should only occur under either of the two if statements.

```py
            if o == "cooldown":  # special edge case, sadly
                 del ovd["cooldown_count"]
                 del ovd["cooldown_period"]
                 c += 1
             elif o in ovd:
                 del ovd[o]
                 c += 1
```